client/my-sites/domains/domain-management/contacts-privacy/index.jsx

```diff
@@ -78,7 +78,7 @@ class ContactsPrivacy extends React.PureComponent {
 							this.props.selectedDomainName
 						) }
 					>
-						{ translate( 'Edit Contact Info' ) }
+						{ translate( 'Edit contact info' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.